guides/common/modules/proc_importing-templates.adoc

```diff
@@ -38,7 +38,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli_Importing_Templat
 
 To use the API, see the xref:api_Importing_Templates_{context}[].
 
-To use Ansible, see the xref:ansible_Importing_Templates_{context}[].
+To use Ansible, see xref:importing-templates-using-ansible[].
```

Comment:
This seems neat. I don't know if it will work for DITA but if it does, it could allow us to store the generic information about a procedure (introduction, prerequisites, context) in a concept-ish module and then point to the various minimal procedures for different tools. Is this how you intended it?

Reply:
I'm not sure it makes sense to put these xrefs under Prerequisites. They're alternative ways to perform the same task.

Reply:
It is like that on "master". -> I moved them above, but everything else is out of scope of this PR.